src/pages/settings/Members.tsx

```diff
@@ -0,0 +1,284 @@
+                    </Typography>
+                  </Button>
+                </Loadmore>
+              )}
+            </>
+          )}
+        </InvitationsListWrapper>
+      )}
+
+      <Head>
+        {!!membersLoading ? (
+          <TitleSkeleton variant="text" height={12} width={160} />
+        ) : (
+          <>
+            <Typography variant="subhead">{translate('text_63208b630aaf8df6bbfb266f')}</Typography>
+            {!hasInvites && (
+              <Button
+                variant="secondary"
+                onClick={() => {
+                  createInviteDialogRef.current?.openDialog()
+                }}
+              >
+                {translate('text_63208b630aaf8df6bbfb265b')}
+              </Button>
+            )}
+          </>
+        )}
+      </Head>
+      {!!membersLoading ? (
+        <LoadingListWrapper>
+          {[1, 2, 3].map((i) => (
+            <MembershipItemSkeleton key={i} />
+          ))}
+        </LoadingListWrapper>
+      ) : (
+        <div>
+          {!!membersError ? (
+            <ErrorPlaceholder
+              noMargins
+              title={translate('text_6321a076b94bd1b32494e9ee')}
+              subtitle={translate('text_6321a076b94bd1b32494e9f0')}
+              buttonTitle={translate('text_6321a076b94bd1b32494e9f2')}
+              buttonVariant="primary"
+              buttonAction={membersRefetch}
+              image={<ErrorImage width="136" height="104" />}
+            />
+          ) : (
+            <InfiniteScroll
+              onBottom={() => {
+                const { currentPage = 0, totalPages = 0 } = membersMetadata || {}
+
+                currentPage < totalPages &&
+                  !membersLoading &&
+                  membersFetchMore({
+                    variables: { page: currentPage + 1 },
+                  })
+              }}
+            >
+              <ListWrapper>
+                {membersData?.memberships.collection.map((membership, i) => (
+                  <MembershipItem
+                    key={`membership-item-${i}`}
+                    ref={revokeMembershipDialogRef}
+                    membership={membership}
+                  />
+                ))}
+              </ListWrapper>
+            </InfiniteScroll>
+          )}
+        </div>
+      )}
+
+      {!!hasInvites && <RevokeInviteDialog ref={revokeInviteDialogRef} />}
+      <RevokeMembershipDialog ref={revokeMembershipDialogRef} />
+      <CreateInviteDialog ref={createInviteDialogRef} />
+    </Page>
+  )
+}
+
+const Page = styled.div`
+  padding: ${theme.spacing(12)};
+`
+
+const Title = styled(Typography)`
+  margin-bottom: ${theme.spacing(2)};
+`
+
+const Subtitle = styled(Typography)`
+  margin-bottom: ${theme.spacing(8)};
+`
+
+const Head = styled.div`
+  height: ${NAV_HEIGHT}px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const TitleSkeleton = styled(Skeleton)`
+  margin: 30px 0;
```

Comment:
You have a preperty `marginBottom` on the skeletons to avoid styling them (plus with a spacing out of the DS)

Reply:
Yeah but in that case I want the margin to be on top also, and the component API does not allows such.

I would either increase the margin bottom from the above component or update the API but I felt it was a lot to do to for not that much advantages. I know we have this rule of DS spacing but happy to discuss about it and change the implem

Reply:
We agree to keep it like that